Readme lacks the build editor step

Repository is incomplete and has missed files.

ERROR in src/app/context-demo/context-demo.ts:3:27 - error TS2307: Cannot find module '../../../ckeditor/build/cksource'.

3 import * as CKSource from '../../../ckeditor/build/cksource';
                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/app/demo-form/demo-form.component.ts:8:27 - error TS2307: Cannot find module '../../../ckeditor/build/cksource'.

8 import * as CKSource from '../../../ckeditor/build/cksource';
                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/app/demo-reactive-form/demo-reactive-form.component.ts:7:27 - error TS2307: Cannot find module '../../../ckeditor/build/cksource'.

7 import * as CKSource from '../../../ckeditor/build/cksource';
                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/app/simple-usage/simple-usage.component.ts:2:27 - error TS2307: Cannot find module '../../../ckeditor/build/cksource'.

2 import * as CKSource from '../../../ckeditor/build/cksource';
                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/app/watchdog-demo/watchdog-demo.ts:3:27 - error TS2307: Cannot find module '../../../ckeditor/build/cksource'.

3 import * as CKSource from '../../../ckeditor/build/cksource';

--- a/README.md
+++ b/README.md
@@ -43,6 +43,12 @@ This repository contains the following code:
 
 ### Testing the component (demo)
 
+First, the CKEditor 5 build should be created of source files:
+
+```bash
+npm run build-ckeditor
+```
+
 To open the demo application using the component, run:
 
 ```bash

